src/PokeList.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 14/11/2018
+ * Time: 11:48
+ */
```

Comment:
Remove this autogenerated comment, it's been removed in the other one

Reply:
done in 79576f6